projects/storefrontstyles/scss/components/checkout/order-confirmation/_order-confirmation-overview.scss

```diff
@@ -31,6 +22,38 @@
       @include media-breakpoint-down(sm) {
         padding: 1.25rem;
       }
+
+      .summary-card,
+      .order-summary-card {
+        @include media-breakpoint-down(sm) {
+          background-color: var(--cx-color-inverse);
+          border-width: 1px;
+          border-style: solid;
+          border-color: var(--cx-color-light);
+          margin: 0.625rem 0;
+        }
+      }
+    }
+
+    .order-summary-container {
```

Comment:
redundant scss example with the order-confirmation-overview.html (template) for breaking changes